control-plane/tests/test_phase50_maintainability_closeout.py

```
@@ -0,0 +1,118 @@
+            "#953",
+            "remaining accepted hotspot",
+            "silent re-growth",
+            "another decomposition decision",
+            "bash scripts/verify-maintainability-hotspots.sh",
+            "bash scripts/test-verify-maintainability-hotspots.sh",
+        ):
+            self.assertIn(required, closeout)
+
+    def test_verifier_reports_only_phase50_accepted_hotspot(self) -> None:
+        result = subprocess.run(
+            ["bash", "scripts/verify-maintainability-hotspots.sh"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)